integration-cli/docker_cli_network_unix_test.go

```diff
@@ -1643,9 +1643,13 @@ func (s *DockerSuite) TestDockerNetworkInternalMode(c *check.C) {
 	nr := getNetworkResource(c, "internal")
 	c.Assert(nr.Internal, checker.True)
 
-	dockerCmd(c, "run", "-d", "--net=internal", "--name=first", "busybox", "top")
+	// Current `busybox:latest` is literally `busybox:uclibc` on amd64 and arm{32,64}, refer to:
+	// https://github.com/docker-library/official-images/blob/c85c21613ee36a8c7f80ea3a78cd373337709d02/library/busybox#L47
+	// `busybox:uclibc` image will result in this test case fail like "ping: bad address"
+	// We narrow down this issue to `busybox:latest` and keep track this with the comment here
+	dockerCmd(c, "run", "-d", "--net=internal", "--name=first", "busybox:glibc", "top")
 	c.Assert(waitRun("first"), check.IsNil)
-	dockerCmd(c, "run", "-d", "--net=internal", "--name=second", "busybox", "top")
+	dockerCmd(c, "run", "-d", "--net=internal", "--name=second", "busybox:glibc", "top")
```

Comment:
If we need the `glibc` version of busybox; should it be included in the frozen images?

Reply:
Ah, absolutely I have no objection to add the `busybox:glibc` into the frozen images if we really have somehow subtle requirements to point the `busybox:latest` to the `uclibc` version meaning this will not be changed in the future. If the answer is yes, then I am glad to add the `glibc` version as frozen image...

Reply:
Hmm, I also remember that the `busybox:latest` point to `uclibc` only happens on `amd64` and `arm{32,64}` architectures, that's why the `powerpc` and `s390x` feel comfortable with this changes 😄 IIRC, @tianon perfer to use `busybox:glibc` instead of `busybox:latest` as the frozen images since he is pretty sure `glibc` version is available on all the necessary architectures.

IMO we can use `glibc` version (not frozen, just as this PR's) as interim solution if no side-effect (e.g, all the CI passed) in case of `latest` --> `uclibc` on `amd64` and `arm64` is volatile, but if there's some unknown reason the `latest` --> `uclibc` is mandatory on `{amd, arm}64`, then we can think about to make `glibc` as frozen image